Updated Beginner's Tour

Xatres opened the pull request:
As requested by SabreCat, I've done some editing on the Beginner's Tour. Mostly I just flesh out the guide to how the game works and try to contextualize everything on the page with the user avatar. If we implement these changes, there may need to be a few edits to the starting task descriptions to prevent overlap.

Definitely check to make sure I didn't mess up any of the coding. I'm a writer, not a programmer.

Xatres committed:
Updated Beginner's Tour

As requested by SabreCat, I've done some editing on the Beginner's Tour. Mostly I just flesh out the guide to how the game works and try to contextualize everything on the page with the user avatar. If we implement these changes, there may need to be a few edits to the starting task descriptions to prevent overlap.

Definitely check to make sure I didn't mess up any of the coding. I'm a writer, not a programmer.

SabreCat commented:
High five! I'll take a look at this sometime this week. Did you see [where the default tasks live](https://github.com/HabitRPG/habitrpg/blob/develop/locales/en/app.json)?

SabreCat commented:
Got to check this out today. The tour stepped through flawlessly. Good friggin' work, man! This is a huge improvement over the functional but stale current tour. So take my following suggestions in that light--these are tweaks I'd like to see, but even if we merged this right now it'd still be fantastic.
- Any way to make the avatar description gender-neutral? It's [always bugged me that we assume a white male player](https://github.com/HabitRPG/habitrpg/issues/1529), and it's my hope that someday there might not be a guarantee that your starting character is male.
- Strike "when the tour is complete" from the Avatar Customization step--you could jump into customization while the tour is underway, if you want.
- I prefer "Health" instead of "HP", and "Experience" instead of "Exp", whenever possible, though we're not always consistent about it.
- Since people can change their "midnight", might it be worth mentioning custom day start? Maybe as a second step on the Dailies column?
- Should be "To-Dos", not "Todos".
- It's not entirely true that To-Dos don't have deadlines; you can set such things in the Due Date field. They're cosmetic/informational for now, though.
- I'd like to see some mention, at the end of the tour, that there are more features the player can unlock as they level up: what you see at the start is not the whole game!

Thanks so much for working on this! I'm looking forward to seeing what you come up with for a revision of the default tasks!

SabreCat commented:
Worth noting on that first bullet that we're stripping out references to gender altogether where possible, per [Tyler's video on sprite updates and customization](http://vimeo.com/81066829).

Xatres committed:
Updated Beginner's Tour, Mk 2

Fixed issues proposed by SabreCat, including gender neutral language, general terminology, and adding a note about unlockable features at the end.

SabreCat commented:
:heart: :heart: :heart:
At each step you revised for this pass, I sighed and said, "beautiful". Love it!

I have only one teensy suggestion left: change the mention of "sex" to "body type". At the moment it's sex/gender, but when Customization changes roll out in the next month or so, we'll have "broad" and "slim" avatars instead of "male" and "female". It'd be nice to have that wording already in place so we don't need to revise the tour when those features deploy.

Tagging @lefnire, this is going to be ready to merge real soon. :+1:

Xatres committed:
Updated Beginner's Tour, Mk 3

Quickfix to changed "sex" to "body type," in anticipation of coming customization updates.

SabreCat commented:
Booyah!

In true QAer fashion, I thought of one more improvement when reviewing your updates. In the final step of the tour, "Unlock New Features," turn the HabitRPG Wiki mention into an actual link:

`<a href="http://habitrpg.wikia.com" target="_blank">HabitRPG Wiki</a>`

Then it'll be perfect, as far as I'm concerned!

Xatres committed:
Updated Beginner's Tour, Mk 4

Added link to HabitRPG wiki.

lefnire commented:
@Xatres you're one patient dude :) I had a good laugh from this thread. What's your UUID, you need a blacksmith level

Xatres commented:
@lefnire My UUID = 0bb883af-b7ed-46e3-bac8-ef867823e886

My work was probably more Scribe than Blacksmith, since I didn't really do any coding. I'd have to seriously brush up on my programming skills before attempting to code anything for the site.

lefnire commented:
√

djuretic commented on public/js/services/guideServices.js:
```diff
@@ -19,35 +19,57 @@ angular.module('guideServices', []).
+        }, {
+          element: "ul.habits li:first-child",
+          title: "Unlock New Features",
+          content: "That's all you need to know for now, but I'll be back as you level up to let you know about new features you've unlocked. Each new feature will give you more incentives to accomplish your goals. Find out more at the <a href="http://habitrpg.wikia.com" target="_blank">HabitRPG Wiki</a> or let yourself be surprised. Good luck!",
```
@Xatres Can you change this:

`<a href="http://habitrpg.wikia.com" target="_blank">HabitRPG Wiki</a>`

with this

`<a href='http://habitrpg.wikia.com' target='_blank'>HabitRPG Wiki</a>`?

Having double quotes inside the `content` string gives a syntax error. Thanks!

lefnire replied:
:+1: I've been looking everywhere for that!

Xatres commented:
I had to submit the fix as a separate pull request, but it's done.

Xatres commented:
Linkage: https://github.com/HabitRPG/habitrpg/pull/1971
